### ISSUE TITLE
Should different rollback tags be kept for different environments?

### ISSUE THREAD

**Issue by benlk** [INN/deploy-tools#27]
Consider the situation where you deploy to production, deploy to staging, and then rollback production. Production's rollback will use staging's rollback tag, right? 

Should the rollback tags be named rollback-environment instead?

**Comment by rnagle** [comment]
Good point. We can probably address this with a prefix for the rollback tag depending on the environment.
